experiment_design_doc.md

```diff
@@ -94,6 +94,16 @@ The experiment infrastructure consists of several components:
   performance of subjects using Tellina versus those who are not, via
   statistical analysis.  This will be done with a post-processing program.
 
+### User Requirements [Remote Edition]
+The subjects will participate to the experiment remotely. Thus, they will not be able to use CSE lab computers and will have to use their laptops.
+
+The participants are required to use a Linux environment. There are three ways to do so:
```

Comment:
Correct me if I'm wrong, but isn't it also possible for them to SSH into a CSE student server? I'm going off of what was available for me when I was a student.

Reply:
It is possible but they also need access to a graphical interface to run meld in a window, which is not possible with a vanilla SSH connection. 

Thinking about it, it could potentially work if we use an x server but then I'm not sure it's easier for them to set up. What do you think?

Reply:
Ah okay. I had misremembered that we supported non-graphical sessions. In which case the three ways are fine. 

If you're interested, there was a discussion on whether we should allow unified diffs in a non-graphical environment (or if the user didn't have Meld installed) and we just decided to disallow it in favor of always having Meld. 

I'll resolve this comment.

Reply:
> If you're interested, there was a discussion on whether we should allow unified diffs in a non-graphical environment (or if the user didn't have Meld installed) and we just decided to disallow it in favor of always having Meld.
Good to know! Has it been recorded somewhere?

Reply:
Not anywhere that I remember. It might be good to document this somewhere in the design doc if you're up to it.

Reply:
Totally if If someone gives me the reasoning in favor of graphical diffs!

Reply:
I think the primary reason was that it maybe hard for newer users to be able to comprehend unified diffs in a terminal. I can't quite remember why else, but personally I also think a graphical diff is significantly more readable than a unified one.

Reply:
Thank you. I will add the rationale in `experiment_design_doc.md`.